readme.md

```diff
@@ -171,8 +171,8 @@ Below is a table showing the mapping of the LTS branches to the packages release
 
```

Comment:
We will also need to update the LTS description above:
A new LTS version will be created every **6 months**. The lifetime of an LTS branch is currently planned for **one year**. LTS branches receive all bug fixes that fall in one of these categories: